bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -148,12 +152,36 @@ <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
     </section>
   </div>
 </div>
+
+{% if l10n_has_tag('fxnew_account_032019') %}
+<div id="firefox-account">
+  <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get everything else Firefox.') }}</h4>
+  <p class="fxa-intro">{{ _('Sync bookmarks, send tabs, save to Pocket and more with a Firefox Account.') }}</p>
+
+  {{ fxa_email_form(entrypoint='firefox-new', button_text=_('Get a Firefox Account'), utm_source='firefox-new', utm_params={'campaign': 'firefox-new', 'content': 'firefox-new-fxa', 'term': 'existing-users', 'medium': 'referral'}) }}
+
+  <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': '/firefox/accounts/', 'source': 'accounts-page'}) }}>{{ _('Sign In') }}</a></p>
+
+  {{ download_firefox(dom_id='download-firefox-account-modal', alt_copy=_('Download Anyway'), button_color='button-hollow button-blue', locale_in_transition=True, download_location='other') }}
```

Comment:
If I hit "Download Anyway" and /thanks/ loads, I'm immediately presented with another signup form for a newsletter. Could we forgo the newsletter signup on /thanks/ in this instance and show the app store badges instead? It feels a little over-pushy being shown two things?